pyproject.toml

```diff
@@ -21,6 +21,7 @@ dependencies = [
     "fastapi>=0.115.12",
     "uvicorn>=0.34.2",
     "prometheus-client>=0.21.1",
+    "aiosqlite>=0.19.0"
```

Comment:
we are using `asyncmy` => mysql and not sqlite, so behaviour may be very different